### DISCUSSION TITLE
Help figuring out stagger settings

### DISCUSSION THREAD

**Discussion by ghost** [ericaltendorf/plotman#706] [Q&A]
Hi,
I have 3 1tb NVME and dual xeon CPUs.
I want to setup the stagger phases so that there 9 plotters total and 3 jobs one major phase apart, 3 jobs per nvme. 
Could anyone be bothered to help me figure this out?

Like so:
  #    plot id   tmp    phase    
  0  2dd687  ssd3   1:1   
  1   b1126     ssd2   1:2  
  2  52a7a     ssd1    1:3   
  3  13140     ssd3   2:4  
  4  1c203     ssd2   2:5  
  5  3f15c      ssd1    2:6  
  6  2341e     ssd3   3:5 
  7  x1d03     ssd2   3:5 
  8  x1d03     ssd2   3:6

**Comment by xetura** [comment]
This should work:        
        
tmpdir_stagger_phase_major: 2
tmpdir_stagger_phase_minor: 1
tmpdir_stagger_phase_limit: 1
tmpdir_max_jobs: 3
